emscripten.py

```diff
@@ -756,7 +756,8 @@ def create_receiving(exports):
       for s in exports_that_are_not_initializers:
         mangled = asmjs_mangle(s)
         dynCallAssignment = ('dynCalls["' + s.replace('dynCall_', '') + '"] = ') if generate_dyncall_assignment and mangled.startswith('dynCall_') else ''
-        receiving += [dynCallAssignment + mangled + ' = asm["' + s + '"];']
+        export_assignment = f'Module["{mangled}"] = ' if settings.MODULARIZE and settings.EXPORT_ALL else ''
+        receiving += [f'{export_assignment}{dynCallAssignment}{mangled} = asm["{s}"]']
```

Comment:
It seems like maybe we a duplicating code from `make_export_wrappers` here.. can we just call it instead?

I'm not also no a big fan of the use of `if else` like this on a single line.  How about:

```
if settings.MODULARIZE and settings.EXPORT_ALL:
  export_assignment = f'Module["{mangled}"] = '
else:
  export_assignment = ''
``

Reply:
For the `if else` on the same line, it was requested in an earlier comment to keep it similar to the code there was before (the `dynCallAssignment`.

I don't know if we want to call `make_export_wrappers` because in the case of `MINIMAL_RUNTIME`, the `var` definition is done ahead of this function. I technically can move this code in the `make_export_wrappers` if you want, and call it with `exports_that_are_not_initializers`.

Reply:
fwiw I think the single-line code is more readable.

Reply:
I do feel a bit more strongly about these.  Personally, I find maintaining code that uses this style a lot harder.  There seem to be general agreement that its bad style here: https://stackoverflow.com/questions/35070436/python-ternary-style-is-this-good-or-bad-style.   

The google style guide does say they are OK in simple cases:  https://google.github.io/styleguide/pyguide.html#2114-decision.   But I would say when the length of the line is over 80 chars, and contains multiple embedded f-strings, we a no longer that catagory.

Reply:
I pushed without any `else` so I think it meets halfway 😄